### ISSUE TITLE
classify specific members in modelBase class

### ISSUE THREAD

**Issue by benMen87** [giladcohen/tensorflow-TB#5]
I think logits and predictions should not be members of ModelBase rather of ClassifierModel.
since they are specific for classification tasks.

**Comment by giladcohen** [comment]
Will be fixed after my vacation. ETA: 8-Sep-2017
